tests/lava/lib/peripherals/realsense/test_realsense.py

```diff
@@ -0,0 +1,108 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+import unittest
+import sys
+from pathlib import Path
+
+from lava.lib.peripherals.realsense.realsense import Realsense
+from lava.magma.core.run_configs import Loihi2SimCfg
+from lava.magma.core.run_conditions import RunSteps
+
+has_valid_camera = False
+try:
+    import pyrealsense2 as rs
+
+    for device in rs.context().devices:
+        color_sensor = device.first_color_sensor()
+        depth_sensor = device.first_depth_sensor()
+
+        if color_sensor is not None and depth_sensor is not None:
+            has_valid_camera = True
+            break
+except ImportError:
+    print("Need `pyrealsense2` library installed.", file=sys.stderr)
+    exit(1)
+
+
+class TestRealsense(unittest.TestCase):
+    @unittest.skipUnless(has_valid_camera, "Requires valid Realsense camera.")
```

Comment:
See above for description (valid -> Requires a Realsense camera to be connected.)

Reply:
Corrected